src/command_modules/azure-cli-container/azure/cli/command_modules/container/_params.py

```diff
@@ -29,13 +29,25 @@ def _environment_variables_type(value):
     return {'name': env_name, 'value': env_value}
 
 
+def _secure_environment_variables_type(value):
+    """Space-separated values in 'key=value' format."""
+    try:
+        env_name, env_secure_value = value.split('=', 1)
+    except ValueError:
+        message = ("Incorrectly formatted secure environment settings. "
+                   "Argument values should be in the format a=b c=d")
+        raise CLIError(message)
+    return {'name': env_name, 'secureValue': env_secure_value}
```

Comment:
Move this to be after line 35.

Reply:
Do you mean add extra spaces above the _secure_environment_variables_type to push it down to line35?

Reply:
Why leave the return statement out of the `try-except` structure? Since this is the only positive exit of the method, you can just leave it in the `try`.

Reply:
That's a good point, I misunderstood.